### PR TITLE
update to the new api

### PR DESCRIPTION
**Pull request by czue**
forgot to PR this earlier. goes with https://github.com/dimagi/pillowtop/pull/180 and https://github.com/dimagi/commcare-hq/pull/9063 

@calellowitz @snopoke

### CODE REVIEW

**Comment by snopoke** [comment]
:+1: will merge all once these tests pass

**Comment by czue** [comment]
sorry accidentally had another change in here. just rebased the change i wanted to PR now

**Comment by czue** [comment]
:white_check_mark: I think all 3 are now good to go unless you have feedback

File: requirements.txt [hunk 1]
```diff
@@ -7,5 +7,5 @@ alembic==0.6.4
 unittest2==0.5.1
 psycopg2>=2.4.1
 -e git+git://github.com/dimagi/fakecouch.git#egg=fakecouch
--e git+git://github.com/dimagi/pillowtop.git#egg=pillowtop
```
Comment (snopoke) [inline requirements.txt]: I don't think this has any effect on anything. Travis uses the requirements listed in `setup.py`

Comment (snopoke) [inline requirements.txt]: Actually I see travis does also install requirements from requirements.txt.

**Pull request merged**